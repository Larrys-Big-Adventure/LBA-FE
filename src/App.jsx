import React, { useState } from 'react';
import { Route, Switch } from 'react-router-dom';
import { Home, Login, NoMatch, Register } from './containers';

import { PrivateRoute } from './components';
import GlobalStyle from './styles';

function App() {
  // const [auth, setAuth] = useState({});
  return (
    <>
      <GlobalStyle />
      <Switch>
        <Route exact path="/" component={Home} />
        <Route path="/login" component={Login} />
        <Route path="/register" component={Register} />
        <Route path="*" component={NoMatch} />
      </Switch>
    </>
  );
}

export default App;
