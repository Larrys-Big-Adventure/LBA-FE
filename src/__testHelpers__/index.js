import render from './render';
import renderer from './renderer';

export { render, renderer };
