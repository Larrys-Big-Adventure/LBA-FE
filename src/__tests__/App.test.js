import React from 'react';
import '@testing-library/jest-dom/extend-expect';
import { cleanup } from '@testing-library/react';
import { render, renderer } from '../__testHelpers__';

import App from '../App.jsx';

afterEach(cleanup);

describe('App', () => {
  it('renders without crashing', () => {
    render(<App />);
  });

  it('matches previous snapshot', () => {
    const tree = renderer(<App />);
    expect(tree.toJSON()).toMatchSnapshot();
  });
});
