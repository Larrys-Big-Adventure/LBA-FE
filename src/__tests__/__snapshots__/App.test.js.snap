// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App matches previous snapshot 1`] = `null`;
